drizzlepac/haputils/catalog_utils.py

```diff
@@ -197,69 +210,99 @@ def compute_background(self, box_size, win_size,
 
         # SExtractorBackground ans StdBackgroundRMS are the defaults
         bkg = None
+        is_zero_background_defined = False
 
         # Make a local copy of the data(image) being processed in order to reset any
         # data values which equal nan (e.g., subarrays) to zero.
         imgdata = self.data.copy()
-        imgdata[np.isnan(imgdata)] = 0
-
-        # Create mask to reject any sources located less than 10 pixels from a image/chip edge
-        # to be used for the sigma-clipping
-        binary_inverted_wht = np.where(imgdata == 0, 1, 0)
-        self.exclude_zones_mask = ndimage.binary_dilation(binary_inverted_wht, iterations=10)
-
-        # Compute a sigma-clipped background which returns only single values for mean,
+        fits.PrimaryHDU(data=imgdata).writeto(self.imgname.replace(".fits", '_imgdata.fits'))
```

Comment:
This line should either be wrapped with logic to only be run in diagnostic mode or flagged for removal before merging.